src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -91947,45 +91947,6 @@ CREATE VIEW VIEW_KNOWN_TRANSPORTER_CONTAINERS AS SELECT
         t.CONTAINER_NAME AS ContainerName,
         t.MEMBRANE AS MembraneLocation
 FROM tab_known_transporters_containers t;
-CREATE VIEW [ContainerParameters_Species]

-AS

-SELECT DISTINCT 

-                [cpc].[container_id], 

-                [cpc].[container_type], 

-                [cpc].[container_name], 

-                [cpc].[parameter_name], 

-                [species]

-FROM   [tab_container_parameter_curves] AS [cpc],

-       [tab_container_parameters] AS [cp]

-WHERE  [cpc].[container_id] = [cp].[container_id]

-         AND [cpc].[parameter_name] = [cp].[parameter_name]

-         AND [building_block_type] = "INDIVIDUAL"

-UNION


```

Comment:
this part was removed from the query; the rest is the same as before
(hard to see in the diff because the view was moved to the end in the new db dump).